lib/agora/agora_dispatcher.ex

```diff
@@ -0,0 +1,91 @@
+defmodule Membrane.Agora.Dispatcher do
+  @moduledoc false
```

Comment:
This should be either used somewhere or documented for public use ;) Also, tests are missing

Reply:
I have added the documentation and the tests, but as you know there are problems with running these integration tests :/